content/updates/2020-07/1-rules-UI.md

```diff
@@ -0,0 +1,26 @@
+---
+pim_announcement_img: ./img/rules_UI.png
+pim_announcement_alt_img: "The UI to manage the rules"
+pim_announcement_audience:
+- EE
+---
+
+# New! Create and update rules from the PIM!
+::: meta-data type="New" features="Rules engine, Productivity" available="early July" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab.html"
```

Comment:
Isn't it "mid-July" instead of early July, @morganen?

Reply:
indeed. It was the 16th